src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -293,12 +358,15 @@ bool mongoConnect(const char* host) {
 * object to be mocked.
 *
 */
-void mongoDisconnect() {
+void mongoDisconnect()
```

Comment:
Ah this one!
Yeah, I remember I removed it by commenting it out.
I'll try to remove this function as well.